spot-ml/ml_test.sh

```diff
@@ -35,6 +35,14 @@ LPATH=${LUSER}/ml/${DSOURCE}/test
 HPATH=${HUSER}/${DSOURCE}/test/scored_results
 # prepare parameters pipeline stages
 
+# pass the user domain designation if not empty
```

Comment:
is this just a merge with another bugfix or is this a bugfix that got tacked onto this feature changeset?